test/vendir/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG IMAGE=containerbase/buildpack
```

Comment:
i don't see a need for this new test fixture. please integrated into latest test fixture.